packages/fluentui/docs/src/prototypes/Calls/recentContact.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Avatar, Flex, Text, Menu } from '@fluentui/react-northstar';
+import { CallIcon, CallVideoIcon } from '@fluentui/react-icons-northstar';
+
+const Contact: React.FC<{
+  image?: string;
+  name: string;
+  status: object;
+}> = ({ image, name, status }) => {
+  return (
+    <Flex fill gap="gap.small" space="between">
+      <div>
```

Comment:
Can we use `Flex` with some `gap` here instead of adding div and overrides on the children for the margins?